packages/rs-drive-abci/src/config.rs

```diff
@@ -670,6 +671,7 @@ impl PlatformConfig {
             tokio_console_retention_secs: PlatformConfig::default_tokio_console_retention_secs(),
             initial_protocol_version: Self::default_initial_protocol_version(),
             prometheus_bind_address: None,
+            // TODO: This is dangerous. The default value must be 127.0.0.1
```

Comment:
Yes, better to use 127.0.0.1

Reply:
you added this comment not me, irrelevant to this pr